src/cache.ts

```diff
@@ -25,7 +25,7 @@ export class ListWatch<T extends KubernetesObject> implements ObjectCache<T>, In
     private objects: T[] = [];
     private resourceVersion: string;
     private readonly indexCache: { [key: string]: T[] } = {};
-    private readonly callbackCache: { [key: string]: (ObjectCallback<T> | ErrorCallback)[] } = {};
+    private readonly callbackCache: { [key: string]: Array<ObjectCallback<T> | ErrorCallback> } = {};
```

Comment:
Could you explain why you need to make these changes in this file and the corresponding test?

Reply:
Functions like `k8sApi.listPodForAllNamespaces()` no longer return the HTTP status and body. They return just the KubernetesObjects. I updated ListPromise to reflect this and updated the tests. Let me know if this is out of line, but the example and tests do not work otherwise.

Reply:
Oh, and the Array updates are linting fixes.